testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -405,6 +407,10 @@
 PRMErr2 = resource_filename(__name__, 'data/Amber/ace_mbondi3.error2.parm7')
 PRMErr3 = resource_filename(__name__, 'data/Amber/ace_mbondi3.error3.parm7')
 
+<<<<<<< HEAD
```

Comment:
It looks like you had a git merge conflict that you did not fix. These lines are not recognised by python and cause issues.

Reply:
Thanks, I fixed and resubmit that